package.json

```diff
@@ -35,14 +35,17 @@
     "dist"
   ],
   "main": "./dist/index.cjs",
-  "exports": [
-    {
-      "import": "./src/index.mjs",
-      "require": "./dist/index.cjs",
-      "default": "./dist/index.cjs"
-    },
-    "./dist/index.cjs"
-  ],
+  "exports": {
+    ".": [
```

Comment:
the dot means "main" here

Reply:
ahhh good point. node 13.2 - 13.6 can, they just will fall back to the string format - but i think node 13.0 and 13.1 won't, and will still require another array fallback?

I'll make a third PR if that's indeed the case, but this is still an improvement as is :-)